app/controllers/concerns/curation_concerns/embargoes_controller_behavior.rb

```diff
@@ -14,10 +14,9 @@ def index
 
     # Removes a single embargo
     def destroy
-      update_files = !curation_concern.under_embargo? # embargo expired
       EmbargoActor.new(curation_concern).destroy
       flash[:notice] = curation_concern.embargo_history.last
-      if update_files
+      if curation_concern.works_generic_work? && curation_concern.generic_files.present?
```

Comment:
I'd rather leave it how it is.  There could be a ticket for DRYing up this at a later date.  It would add a method to FileSet that has no other use case currently.  @terrellt can we punt on this line?